src/Templating/Helper/BaseHelper.php

```diff
@@ -28,8 +27,13 @@
  *
  * @author Alexander <iam.asm89@gmail.com>
  */
-abstract class BaseHelper extends Helper
+abstract class BaseHelper
```

Comment:
WDYT about this BC break @sonata-project/contributors . The methods added by this extends are on this class now, so it is only a BC break for those checking what classes does this extend or implemented interface.

IMO if everything works, it is fine. That helper does not bring anything, it is just a get/set

Reply:
NVM, let's add first support for SF 5 and twig 3, without BC breaks. This is not needed.